src/imagelightbox.js

```diff
@@ -453,13 +407,13 @@
             },
 
             _loadPreviousImage = function () {
-                if (options.previousTarget() !== false) {
```

Comment:
Sure, but remember, these functions are now replaced by events! This could probably be cleared up in the docs as this could represent a breaking change for some people. That said, using this options should have caused things in the first place. I think.